allennlp/modules/feedforward.py

```diff
@@ -79,6 +80,8 @@ def forward(self, inputs: torch.Tensor) -> torch.Tensor:
             output = dropout(activation(layer(output)))
         return output
 
+    # Requires custom logic around the activations (mostly because the Union type is too complex
+    # to handle automatically).
```

Comment:
Ah, yes, we could handle the `List`, I think, as I mentioned above.  But, yeah, that can wait, as it's not very frequent, and not a big deal.  You might modify the wording here, though - it's not too complex to handle automatically, it's just complex enough that we haven't bothered to do it yet.

Reply:
More specifically, it's because the `Union` isn't a `Union` of native types, right? I tried this with the `Maxout` module and it worked fine with lists of ints.

Reply:
the from_params logic will as a last resort pass the values on as-is, so it works if it's a union of native types *and also* the instantiation logic doesn't vary depending on what you pass in